sympy/simplify/sqrtdenest.py

```diff
@@ -236,6 +236,18 @@ def _sqrtdenest0(expr):
         else:
             n, d = [_sqrtdenest0(i) for i in (n, d)]
             return n/d
+
+    if isinstance(expr, Add):
+        cs = []
+        args = []
+        for arg in ordered(expr.args):
```

Comment:
In general, the resulting order will be different from the order used by Add, for example,
```python
>>> e = -6*sqrt(55)-6*sqrt(35)-2*sqrt(22)-2*sqrt(14)+2*sqrt(77)+6*sqrt(10)+65
>>> e.args
(65, -6*sqrt(35), -6*sqrt(55), -2*sqrt(14), -2*sqrt(22), 2*sqrt(77), 6*sqrt(10))
>>> tuple(ordered(e.args))
(65, 6*sqrt(10), -2*sqrt(14), -2*sqrt(22), -6*sqrt(35), -6*sqrt(55), 2*sqrt(77))
```